### PR TITLE
Fix and standardize app docs headers

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request fixes apps docs headers -- 
- sentence case
- Sample _queries_ and Sample _logs_ plural for uniformity

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md [hunk 1]
```diff
@@ -42,18 +42,18 @@ Watch the following micro lesson to learn about Cloud Infrastructure Security fo
...
-* [Amazon CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html) 
+* [Amazon CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html)
 * [Amazon GuardDuty](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings.html)
 * [AWS Network Firewall](https://docs.aws.amazon.com/network-firewall/latest/developerguide/what-is-aws-network-firewall.html)
 * [AWS Security Hub](https://docs.aws.amazon.com/securityhub/latest/userguide/what-is-securityhub.html)
-* [AWS Web Application Firewall (WAF)](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html) 
+* [AWS Web Application Firewall (WAF)](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html)
 
-## Sample log message
+## Sample log messages
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:51]: I know you're just trying to standardize, but why would you change "Sample log message" to "Sample log messages" when there is only one log message shown?

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Cloud Infrastructure Security for AWS utilizes the following log types:
 }
 ```
 
-## Sample query
+## Sample queries
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:82]: Similar question as above. Why would you change "Sample query" to "Sample queries" when there is only one query shown?

**Review by jpipkin1 — APPROVED** [review]
Approved, @kimsauce. I didn't really look at this one too carefully, since you were making the same changes repeatedly.

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md (hunk 2 above)
Comment (kimsauce) [reply docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:82]: I was hoping we could treat those headers as UI (they appear in every single apps doc) and keep them uniform. Here's an example -- these docs say "Endpoints" even if there's only one.

Screenshot 2024-01-23 at 5 13 27 PM

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md (hunk 1 above)
Comment (kimsauce) [reply docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:51]: https://github.com/SumoLogic/sumologic-documentation/pull/3558#discussion_r1464183159

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md (hunk 2 above)
Comment (jpipkin1) [reply docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:82]: Makes sense to me, @kimsauce.

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md (hunk 1 above)
Comment (jpipkin1) [reply docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:51]: Okay, @kimsauce. That makes sense.

**Pull request merged**